### DISCUSSION TITLE
Track SDL3's Sources by date

### DISCUSSION THREAD

**Discussion by felipetesc** [revmischa/sdl3-rs#12] [Ideas]
Hello. Currently, [SDL]( https://github.com/libsdl-org/SDL ) states that its version 3 is only available at the main branch. This is something bad if want to make the bindings related to the current version, because things are changing alot. So I want to give the idea to track this crate's version to a specific date time of SDL3. When they finally start given versions to SDL3, maybe changing the approach from date time to SDL3 version. Perhaps a new entry to cargo which could be used to point to the SDL's source code could be used, and when SDL3 gets its own version, we track sdl3-rs to SDL3's version.
What do you think about that ?

**Comment by revmischa** [comment]
Definitely agree - right now in the sdl3-sys subdirectory we have a git submodule pointing at a specific commit of SDL3. We can use that to track what version of SDL3 we're building against.

**Comment by felipetesc** [comment]
Hi. First SDL3 preview is out : https://github.com/libsdl-org/SDL/tags

**Reply by revmischa** [reply]
Yep thanks! We're working on migrating to the new version. Contributions welcome!
It's based on this [sdl3-sys](https://github.com/maia-s/sdl3-sys-rs)
